oe_contact_forms.module

```diff
@@ -537,3 +539,23 @@ function oe_contact_forms_mail_alter(&$message) {
   // Change subject for corporate mail, copy and reply.
   $message['subject'] = $contact_form->getThirdPartySetting('oe_contact_forms', 'email_subject');
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_access().
+ */
+function oe_contact_forms_contact_form_access(EntityInterface $entity, $operation, AccountInterface $account) {
```

Comment:
Not good. As discussed, we need to check create on message not form: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21entity.api.php/function/hook_ENTITY_TYPE_create_access/8.2.x. See my comment in the jira ticket as well.

Reply:
ok, makes sense